network/address.go

```diff
@@ -196,6 +198,10 @@ func (a Address) Valid() bool {
 		return false
 	}
 
+	// If hostname is missing then they mean "all IPs", and that's valid.
+	if len(ip) == 0 {
+		return true
+	}
```

Comment:
Can't seem to make a comment on unchanged lines. But the documentation for `Valid()` seem to be wrong. The example should be `tls://192.168.1.10:5678` and not `tls:192.168.1.10:5678`.